vector-app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-sdk tools:overrideLibrary="com.sayem.keepawake,com.splashview,com.facebook.react,com.facebook.hermes" />
+
```

Comment:
The existing `tools:overrideLibrary` is actually located here: https://github.com/element-hq/element-android/blob/main/vector/src/main/AndroidManifest.xml#L57.

Better to add new override there.